### PR TITLE
kubespy: Add version 0.6.3

### PR DESCRIPTION
**Pull request by cowwoc**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by aliesbelik** [comment]
As a non-GUI tool could be submitted into the `main` bucket instead, since meeting the [Main criteria](https://github.com/ScoopInstaller/Scoop/wiki/Criteria-for-including-apps-in-the-main-bucket).

**Comment by aliesbelik** [comment]
Please add arm64 version and hash autoupdate section as well.

**Comment by cowwoc** [comment]
@aliesbelik Thanks for the head's up regarding the `main` bucket. I didn't know it was an option.

Regarding the hash, does `GitHub` post sha256 values for all downloads? I couldn't find any reference for this...

File: bucket/kubespy.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            "hash": "3a3b266808b15aea79b9a71666a743d90dd976ef17ce12b9feee9e5c303d3a7f"
+        }
+    },
+    "bin": "kubespy.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/pulumi/kubespy/releases/download/v$version/kubespy-v$version-windows-amd64.tar.gz"
+            }
```
Comment (aliesbelik) [inline bucket/kubespy.json:18]:
```suggestion
            },
            "arm64": {
                "url": "https://github.com/pulumi/kubespy/releases/download/v$version/kubespy-v$version-windows-arm64.tar.gz"
            }
```

File: bucket/kubespy.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        }
```
Comment (aliesbelik) [inline bucket/kubespy.json:19]:
```suggestion
        },
        "hash": {
            "url": "$baseurl/checksums.txt"
        }
```

File: bucket/kubespy.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.6.3",
+    "description": "A tool for observing Kubernetes resources in real time.",
+    "homepage": "https://github.com/pulumi/kubespy",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/pulumi/kubespy/releases/download/v0.6.3/kubespy-v0.6.3-windows-amd64.tar.gz",
+            "hash": "3a3b266808b15aea79b9a71666a743d90dd976ef17ce12b9feee9e5c303d3a7f"
+        }
```
Comment (aliesbelik) [inline bucket/kubespy.json:10]:
```suggestion
        },
        "arm64": {
            "url": "https://github.com/pulumi/kubespy/releases/download/v0.6.3/kubespy-v0.6.3-windows-arm64.tar.gz",
            "hash": "d0bb9ed65b4b5869280e6030058c8eed32d4f6590ceff666cf4221429745ea84"
        }
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]
Adding arm64 version and hash autoupdate.

**Comment by cowwoc** [comment]
Thank you!

**Comment by cowwoc** [comment]
@aliesbelik Should I close this PR and open a new one against the main bucket?

**Review by aliesbelik — APPROVED** [review]

**Comment by aliesbelik** [comment]
> Should I close this PR and open a new one against the main bucket? Or are we able to move it over somehow?

I'm not maintainer of Main so I couldn't transfer your request.
If you feel enthusiastic enough, please re-create PR with final version of manifest in Main.
Otherwise looks good and could be merged into Extras.

**Comment by cowwoc** [comment]
Let's leave it in extra. You can go ahead with the merge. Thanks again for your help!

**Pull request merged**